.github/workflows/daily.yml

```diff
@@ -0,0 +1,41 @@
+name: Daily
+
+# See https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule
+
+on:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron:  '30 5 * * *'
+
+jobs:
+  validate-cclw-sheet:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Use .env.example
+        run:  cp .env.example .env
+
+      - name: Get dependencies  
+        run: pip install --upgrade google-api-python-client google-auth-httplib2 google-auth-oauthlib
+
+      - name: Download Sheet
+        run: GOOGLE_SERVICE_ACCOUNT='${{ secrets.GOOGLE_SERVICE_ACCOUNT }}' python scripts/validate_cclw_sheet/download.py > download.csv
+
+      - name: Get python Container
```

Comment:
Could the get python container and build steps be replaced by a docker build {path to backend Dockerfile}?

Just as from my understanding the base image will be pulled anyway and we are only building the backend image from the images in the docker-compose?

Reply:
I was thinking of something similar - but ended up just doing what is normally done on main - but you may be right, we could pull the tag `main-xxxx` where xxxx is the short commit hash on main ... certainly would be an improvement